docs/openapi.yaml

```diff
@@ -536,13 +600,52 @@ components:
           type: array
           items:
             type: string
-            example: "1ab34d9c"
+            example: 1ab34d9c
+    Payment:
+      type: object
+      required:
+        - paid_by
+        - amount
+        - split_amongst
```

Comment:
Thats because the values are generated with the jsonify function in the models.py, having variable values for transaction or payments would require to distinguish them inside the jsonify function or rewrite their values.